contrib/libdivide-cmake/libdivide-config.h

```diff
@@ -0,0 +1,15 @@
+#if defined(__SSE2__)
```

Comment:
imo we're better to preserve, just in case, these macros mutually exclusive

Reply:
I think it would be just simpler to have only one macro defined, will be no need to check anything.
anyway we need to rerun checks to see arm perf test results

Reply:
Ok, rebased.

Thought I still don't see any issues here, since types are different.